allure-pytest-bdd/src/pytest_bdd_listener.py

```diff
@@ -41,6 +42,7 @@ def pytest_bdd_before_scenario(self, request, feature, scenario):
             test_result.fullName = full_name
             test_result.name = name
             test_result.start = now()
+            test_result.historyId = md5(request.node.nodeid)
```

Comment:
I use the uuid as the the historyId
test_result.historyId = uuid

Reply:
use nodeid in this case is better